pysrc/utils/descriptive_statistics.py

```diff
@@ -0,0 +1,37 @@
+import pandas as pd
+from ..models.column_statistics import ColumnStatistic
+from ..models.column_information import ColumnInformation
+
+def file_column_statistics(file_path: str, delimiter: str =';', escapechar: str ='\\', is_baseline: bool = False) -> list[ColumnStatistic]:
+    descriptions = []
+    df = pd.read_csv(file_path, delimiter=delimiter, escapechar=escapechar, dtype=str, on_bad_lines='skip')
+    
+    for col_index, column in enumerate(df.columns):
+        column_series = df[column].copy(deep=True)
+        
+        column_series.sort_values(inplace=True, ascending=True)
+        count = column_series.count()
+        unique_count = len(column_series.unique())
```

Comment:
When running this on SCOP, I find some columns where `unique_count > count` holds. See the following log (prints of respective lines added by myself):

```
Greater than one for ColumnStatistic(column_Information=DESCRIPTION.column4, count=70858, unique_count=70859, unique_ratio=1.00001, min='d101m__', max='ds046__', shortest='d101m__', longest='ds046__')!
Greater than one for ColumnStatistic(column_Information=HIERARCHIE.column3, count=94440, unique_count=94441, unique_ratio=1.00001, min='100000', max='99999', shortest='51925', longest='100000')!
outputTime: 3
(metanome-cli) INFO     Elapsed time: 0:00:03.458 (3458 ms).
Greater than one for ColumnStatistic(column_Information=DESCRIPTION.column1, count=7034, unique_count=7035, unique_ratio=1.00014, min='d109m__', max='d9ximd_', shortest='d109m__', longest='d9ximd_')!
Greater than one for ColumnStatistic(column_Information=HIERARCHIE.column1, count=9558, unique_count=9559, unique_ratio=1.0001, min='100006', max='99984', shortest='51156', longest='100006')!
outputTime: 3
(metanome-cli) INFO     Elapsed time: 0:00:00.316 (316 ms).
Greater than one for ColumnStatistic(column_Information=DESCRIPTION.column1, count=707, unique_count=708, unique_ratio=1.00141, min='d112l__', max='d9ilba_', shortest='d112l__', longest='d9ilba_')!
Greater than one for ColumnStatistic(column_Information=HIERARCHIE.column1, count=941, unique_count=942, unique_ratio=1.00106, min='100022', max='99959', shortest='52366', longest='100022')!
outputTime: 4
(metanome-cli) INFO     Elapsed time: 0:00:00.122 (122 ms).
Greater than one for ColumnStatistic(column_Information=DESCRIPTION.column1, count=71, unique_count=72, unique_ratio=1.01408, min='d1a0yb_', max='d7tlia_', shortest='d1a0yb_', longest='d7tlia_')!
Greater than one for ColumnStatistic(column_Information=HIERARCHIE.column1, count=96, unique_count=97, unique_ratio=1.01042, min='101090', max='96432', shortest='53109', longest='101090')!
outputTime: 4
(metanome-cli) INFO     Elapsed time: 0:00:00.052 (52 ms).
Greater than one for ColumnStatistic(column_Information=DESCRIPTION.column1, count=6742, unique_count=6743, unique_ratio=1.00015, min='d101m__', max='d9icya3', shortest='d101m__', longest='d9icya3')!
Greater than one for ColumnStatistic(column_Information=HIERARCHIE.column1, count=9718, unique_count=9719, unique_ratio=1.0001, min='100016', max='99884', shortest='46785', longest='100016')!
outputTime: 3
(metanome-cli) INFO     Elapsed time: 0:00:00.293 (293 ms).
Greater than one for ColumnStatistic(column_Information=DESCRIPTION.column1, count=881, unique_count=882, unique_ratio=1.00114, min='d101m__', max='d7hbib_', shortest='d101m__', longest='d7hbib_')!
Greater than one for ColumnStatistic(column_Information=HIERARCHIE.column1, count=1146, unique_count=1147, unique_ratio=1.00087, min='100067', max='99445', shortest='15492', longest='100067')!
outputTime: 4
(metanome-cli) INFO     Elapsed time: 0:00:00.113 (113 ms).
Greater than one for ColumnStatistic(column_Information=DESCRIPTION.column1, count=93, unique_count=94, unique_ratio=1.01075, min='d1a01a_', max='d4hhbc_', shortest='d1a01a_', longest='d4hhbc_')!
Greater than one for ColumnStatistic(column_Information=HIERARCHIE.column1, count=65, unique_count=66, unique_ratio=1.01538, min='15149', max='92037', shortest='15149', longest='92037')!
outputTime: 4
(metanome-cli) INFO     Elapsed time: 0:00:00.055 (55 ms).
Greater than one for ColumnStatistic(column_Information=DESCRIPTION.column1, count=7066, unique_count=7067, unique_ratio=1.00014, min='d104la_', max='d9ximc_', shortest='d104la_', longest='d9ximc_')!
Greater than one for ColumnStatistic(column_Information=HIERARCHIE.column1, count=9473, unique_count=9474, unique_ratio=1.00011, min='100004', max='99999', shortest='52194', longest='100004')!
outputTime: 3
(metanome-cli) INFO     Elapsed time: 0:00:00.313 (313 ms).
Greater than one for ColumnStatistic(column_Information=DESCRIPTION.column1, count=709, unique_count=710, unique_ratio=1.00141, min='d107m__', max='d9icpa4', shortest='d107m__', longest='d9icpa4')!
Greater than one for ColumnStatistic(column_Information=HIERARCHIE.column1, count=944, unique_count=945, unique_ratio=1.00106, min='100015', max='99936', shortest='53604', longest='100015')!
outputTime: 4
(metanome-cli) INFO     Elapsed time: 0:00:00.123 (123 ms).
Greater than one for ColumnStatistic(column_Information=DESCRIPTION.column1, count=71, unique_count=72, unique_ratio=1.01408, min='d1a67__', max='d7icga3', shortest='d1a67__', longest='d7icga3')!
Greater than one for ColumnStatistic(column_Information=HIERARCHIE.column1, count=89, unique_count=90, unique_ratio=1.01124, min='102059', max='98782', shortest='48088', longest='102059')!
```

I have no clue what could be going wrong there, but something very clearly is incorrect.

Reply:
Fixed by replacing `len(column_series.unique())` with `column_series.nunique()`